### PR TITLE
Use "ISO" publisher when publisher omitted

### LINKED ISSUE #78: If `publisher` is not provided, default publisher is "ISO"
Must have a default publisher of "ISO" if not specified.
```ruby
```ruby
irb(main):005:0> Pubid::Iso::Identifier.new(number: 17301, part: 1).to_s
=> " 17301-1"
```

The result should be "ISO 17301-1".

### PR DESCRIPTION
**Pull request by mico**
closes #78

### CODE REVIEW

File: spec/pubid_iso/identifier_spec.rb [hunk 1]
```diff
@@ -735,5 +735,13 @@
         end
       end
     end
+
```
Comment (ronaldtse) [inline spec/pubid_iso/identifier_spec.rb:738]: Can we add a test for default publisher? Thanks.

**Review by ronaldtse — APPROVED** [review]

File: spec/pubid_iso/identifier_spec.rb (hunk 1 above)
Comment (mico) [reply spec/pubid_iso/identifier_spec.rb:738]: > Can we add a test for default publisher? Thanks.

It's here already https://github.com/metanorma/pubid-iso/pull/86/files/5b135efca112aa562ab29de8d8a30ec6657c0932#diff-59800c3156cfaca079c88759e3ef3df7589c552a11991aa5270988ee28d0a57bL705

Comment (mico) [reply spec/pubid_iso/identifier_spec.rb:738]: > Can we add a test for default publisher? Thanks.

It's on line 705:
image

Comment (ronaldtse) [reply spec/pubid_iso/identifier_spec.rb:738]: But that is really about IDF, I want a test that only tests "default publisher".

Comment (ronaldtse) [reply spec/pubid_iso/identifier_spec.rb:738]: A unit test is supposed to test a singular function on a single case. I prefer not using composite tests to cover multiple features at once, those are supposed to test "integration".

**Pull request merged**